applications/cheersland.md

```diff
@@ -214,6 +223,16 @@ Source codes will reside in: https://github.com/cheersland
 * **FTE:**  4
 * **Costs:** $4,000 DAI
 
+**Goal - Implement Multi-asset Staking Platform and the second game.**  
+We will develop the multi-asset staking module and deploy the Staking Contract on Plasm. When users stake tokens as required, the contract will count HPs based on their staking status. All contracts will be released and used after the audit is completed. We will develop the Multi-asset Staking Platform and open several staking pools including DOT and KSM first. Players can earn HPs by staking a certain amount of DOT or KSM and use them in the game.  
+Our game dev team will develop a new classic game for the Polkadot ecosystem, similar to the Crypto Pirates deployed on BSC, which will adopt the same business model and anti-cheating algorithm.  
+
+We will optimize UI and allow players to have a more cheerful gaming experience.  
+**Therefore, under this milestone, our deliverables include:**  
+- Multi-asset Staking Contract 
+- Multi-asset Staking Platform supporting staked asset such as DOT and kSM 

```

Comment:
How is this different from the previous point? what do you mean by `platform` here - a user facing portal?

Reply:
Yeah sir, this platform is an independent web page that interacts with smart contracts. It supports multi-asset staking in the Polkadot ecosystem. Also, with the increase of game features and users, the platform will open more available staking methods to allow users to have a more pleasant gaming experience and meet other staking needs in the Polkadot ecosystem.